_config.yml

```diff
@@ -17,11 +17,11 @@ gems:
 title: 'Druid'
 description: 'Real²time Exploratory Analytics on Large Datasets'
 
-druid_version: 0.8.2
+druid_version: 0.8.3
 
-druid_rc_version: 0.8.3-rc4
-druid_stable_version: 0.8.2
-druid_latest_version: 0.8.2
+# druid_rc_version: 0.8.3-rc4
```

Comment:
I think it's fine based on https://github.com/druid-io/druid-io.github.io/commit/9068f37be13290334008b3b37011605975d94c11